File: yukicoder/395_永遠の17歳/Main.java
import java.util.Scanner;

public class Main {

    public static void main(String[] args) {

        Scanner sc = new Scanner(System.in);

        int AGE = sc.nextInt();

        if (15 <= AGE) {
            System.out.println(AGE - 7);
        } else {
            System.out.println(-1);
        }

    }

}

File: yukicoder/395_永遠の17歳/Main.cpp
#include <iostream>

using namespace std;

int main() {

    int AGE;
    cin >> AGE;

    if (15 <= AGE) {
        cout << AGE - 7 << endl;
    } else {
        cout << -1 << endl;
    }

    return 0;

}
